configs/common/packages.yaml

```diff
@@ -29,7 +29,8 @@
     cdo:
       version: [2.0.5]
     cmake:
-      version: [3.22.1]
+      version: [3.23.1]
```

Comment:
The changes in this file are needed to avoid duplicate packages being built for our complex environments (`skylab-dev`, `jedi-ufs-all`).